Sources/AppStoreConnectCLI/Files/ResourceFolderManager.swift

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+
+protocol ResourceFolderManager {
```

Comment:
Like many developers I have grown an aversion to the work 'Manager' how about `ResourceProcessor`

Also I feel the function name `read` pairs better with the word `write` as opposed to `save` what do you think?